docs/reference/glossary.rst

```diff
@@ -0,0 +1,23 @@
+Open edX Events Glossary
+##########################
+
+An event has multiple components that are used to define, trigger, and handle the event. This glossary provides definitions for some of the terms to ease the adoption of the Open edX Events library.
+
+.. glossary::
+
+    Event Receiver
+      An event receiver, handler or listener is a function that listens for a specific event and executes custom logic in response to the event being triggered. Since Events are Django-signals, then receivers are registered with the signal dispatcher and are called when the event is emitted. In Django, event receivers are known as signal handlers. Both terms are used interchangeably.
+
+    Event Trigger
+      An event trigger is the action that causes an event to be emitted. When a trigger action occurs, the associated event is emitted, and any registered event receivers are called to handle the event.
+
+    Event Payload
+      The event payload is the data associated with an event that is passed to event receivers when it's triggered. The payload of an event are data attribute classes (e.g. ``CourseEnrollmentData``, ``UserData``, etc.) that carry data about the event such as the event name, timestamp, and any additional metadata and information about the actual event. For more information, see the `Events Payload ADR`_.
```

Comment:
I really like that you linked to an ADR here! I'd recommend linking to others where appropriate.

Reply:
Will do!